modules/graphics/weston.ini.nix

```diff
@@ -48,6 +48,11 @@
       path = "${gala-app}/bin/gala --enable-features=UseOzonePlatform --ozone-platform=wayland";
       icon = "${gala-app}/gala/resources/icon-24x24.png";
     }
+
+    {
+      path = "${pkgs.zathura}/bin/zathura";
+      icon = "${./assets/zathura-icon-24x24.png}";
```

Comment:
About this icon, could we just use one of the icons from${pkgs.zathura}/share/icons/

Reply:
No, because it's svg. I tried -- doesn't work. So I was forced to my png myself

Reply:
Maybe there is a way to make that launcher to show .svg as well? I guess, Zathura won't be the last app with .svg icon

Reply:
Hm, right, another dirs has png. Ok, I will try it

Reply:
```
      icon = "${.pkgs.zathura}/share/icons/hicolor/32x32/apps/org.pwmt.zathura.png";
```
into this, and this is fine

Reply:
Ready